packages/flutter/lib/src/services/text_formatter.dart

```diff
@@ -16,7 +16,7 @@ import 'text_input.dart';
 ///
 /// Concrete implementations [BlacklistingTextInputFormatter], which removes
 /// blacklisted characters upon edit commit, and
-/// [WhitelistingTextInputFormatter], which only allows entries of whitelisted
+/// [WhitelistingTextInputFormatter], which only allows entries of white-listed
```

Comment:
I think we should leave these alone too.
https://www.dictionary.com/browse/whitelist?s=t
https://en.wikipedia.org/wiki/Whitelisting

I think treating it as a single non-hyphenated word also would be more consistent with WhitelistingTextInputFormatter.

Furthermore, "blacklist" is left alone elsewhere.

Reply:
OK, sure.  Reverted.